microsoft/testsuites/libvirt/libvirt_tck.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.testsuite import TestResult
+from lisa.tools import Lscpu
+from lisa.util import SkippedException
+from microsoft.testsuites.libvirt.libvirt_tck_tool import LibvirtTck
+
+
+@TestSuiteMetadata(
+    area="libvirt",
+    category="community",
+    description="""
+    Runs the Libvirt TCK (Technology Compatibility Kit) tests.
```

Comment:
it is better to add brief description what the benchmark does

Reply:
Added.